app/javascript/packs/application.js

```diff
@@ -0,0 +1,17 @@
+// This file is automatically compiled by Webpack, along with any other files
+// present in this directory. You're encouraged to place your actual application logic in
+// a relevant structure within app/javascript and only use these pack files to reference
+// that code so it'll be compiled.
+
+require("@rails/ujs").start()
```

Comment:
in css file and js file, you should use single quote
`"` to `'`

Reply:
done

Reply:
so should i apply single quotes for all css and js file or jus this current js file